.evergreen/build-manylinux.sh

```diff
@@ -39,7 +36,6 @@ ls dist
 # Check for any unexpected files.
 unexpected=$(find dist \! \( -iname dist -or \
                              -iname '*cp27*' -or \
-                             -iname '*cp34*' -or \
```

Comment:
You'll also need to remove 3.4 from build-mac.sh and build-windows.sh. That last one lists the version as "34". You'll probably want to search around for that more broadly.

Reply:
Done